### ISSUE TITLE
Minor linting warnings and errors

### ISSUE THREAD

**Issue by bielsnohr** [stephen-dixon/python-intermediate-inflammation#10]
A number of files across the project could use some linting checks. A lot of these are due to line length, so  perhaps consider defining what the line length used in this project is (and then set it in a .flake8 config file, or whichever linting tool you prefer)

```
inflammation/views.py:4:1: F401 'numpy as np' imported but unused
inflammation/models.py:5:80: E501 line too long (82 > 79 characters)
inflammation/models.py:5:83: W291 trailing whitespace
inflammation/models.py:6:67: W291 trailing whitespace
inflammation/models.py:25:80: E501 line too long (86 > 79 characters)
inflammation/models.py:35:80: E501 line too long (86 > 79 characters)
inflammation/models.py:44:80: E501 line too long (86 > 79 characters)
inflammation/models.py:93:15: E271 multiple spaces after keyword
inflammation/serializers.py:4:1: E302 expected 2 blank lines, found 1
inflammation/serializers.py:44:80: E501 line too long (83 > 79 characters)
inflammation/serializers.py:52:80: E501 line too long (94 > 79 characters)
tests/test_serializer.py:7:80: E501 line too long (82 > 79 characters)
tests/test_serializer.py:8:80: E501 line too long (80 > 79 characters)
tests/test_serializer.py:21:80: E501 line too long (80 > 79 characters)
tests/test_models.py:36:9: F841 local variable 'error_expected' is assigned to but never used
tests/test_models.py:108:80: E501 line too long (91 > 79 characters)
tests/test_models.py:110:80: E501 line too long (87 > 79 characters)
tests/test_models.py:110:88: W292 no newline at end of file
tests/test_patient.py:70:1: W391 blank line at end of file
inflammation-analysis.py:2:80: E501 line too long (96 > 79 characters)
inflammation-analysis.py:34:80: E501 line too long (101 > 79 characters)
inflammation-analysis.py:40:80: E501 line too long (101 > 79 characters)
inflammation-analysis.py:43:80: E501 line too long (86 > 79 characters)
inflammation-analysis.py:47:1: E303 too many blank lines (3)
inflammation-analysis.py:48:80: E501 line too long (103 > 79 characters)
inflammation-analysis.py:50:80: E501 line too long (92 > 79 characters)
```

**Comment by stephen-dixon** [comment]
estimate 5 mins

**Comment by jg3176** [comment]
Addressed in PR #18
